.travis.yml

```diff
@@ -6,10 +6,10 @@ python:
   - "3.5"
   - "3.6"
 install:
-  - "pip install pytest-cov"
-  - "pip install -e ."
+  - "pip install -e .[test]"
```

Comment:
Less Travis config is better, and "test" as name for a set of extra testing requirements is fairly standard, see https://github.com/pypa/sampleproject/blob/master/setup.py.